Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -285,20 +244,6 @@ import Foundation
         )
     }
 
-    // MARK: Analytics Internal Methods
-
-    // TODO: - Remove once all POSTs moved to Encodable
-    func metadataParametersWith(_ parameters: [String: Any]? = [:], for httpType: BTAPIClientHTTPService) -> [String: Any]? {
```

Comment:
Verified this logic still works as expected based on `BTClientMetadata` and the tests in `BTClientMetadata_Tests`